.phan/stubs/wordpress-constants.php

```diff
@@ -24,11 +23,15 @@
 define( 'ABSPATH', './' );
 define( 'WP_DEBUG', (bool) $v );
 define( 'WP_DEBUG_LOG', (bool) $v );
-define( 'WP_PLUGIN_DIR', './' );
-define( 'WPMU_PLUGIN_DIR', './' );
-define( 'EMPTY_TRASH_DAYS', 30 * 86400 );
 define( 'SCRIPT_DEBUG', (bool) $v );
-define( 'WP_LANG_DIR', './' );
+define( 'WPINC', 'wp-includes' );
+define( 'WP_CONTENT_DIR', ABSPATH . 'wp-content' );
+define( 'WP_PLUGIN_DIR', WP_CONTENT_DIR . '/plugins' );
+define( 'WPMU_PLUGIN_DIR', WP_CONTENT_DIR . '/mu-plugins' );
+define( 'WP_LANG_DIR', WP_CONTENT_DIR . '/languages' );
+define( 'WP_CONTENT_URL', 'http://example.com/wp-content' );
+define( 'WP_PLUGIN_URL', WP_CONTENT_URL . '/plugins' );
+define( 'WPMU_PLUGIN_URL', WP_CONTENT_URL . '/mu-plugins' );
```

Comment:
Typically people should use [`plugins_url()`](https://developer.wordpress.org/reference/functions/plugins_url/) instead.